### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
```
Comment (taraselnik) [inline src/components/CommentInfo/CommentInfo.tsx:20]: you can do it without parentheses if you want)

File: src/components/PostInfo/PostInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+      </>
+    );
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
```
Comment (taraselnik) [inline src/components/PostInfo/PostInfo.tsx:33]: same here)

**Review by taraselnik — APPROVED** [review]
enjoy

File: src/App.tsx [hunk 3]
```diff
@@ -1,109 +1,37 @@
...
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(a => a.id === userId);
```
Comment (denys-danyliuk) [inline src/App.tsx:14]:
```suggestion
  return usersFromServer.find(a => a.id === userId);
```

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**